packages/cli/src/scaling/pubsub/pubsub.types.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Pubsub channel used by scaling mode:
+ *
+ * - `n8n.commands` for messages sent by a main process to command workers or other main processes
+ * - `n8n.worker-response` for messages sent by workers in response to commands from main processes
+ */
+export type PubSubChannel = 'n8n.commands' | 'n8n.worker-response';
```

Comment:
Maybe

```suggestion
export type ScalingPubSubChannel = 'n8n.commands' | 'n8n.worker-response';
```
to be more explicit? In case we have other pub sub channels?

Also we could derive the types from the constants in `redis-constants.ts`, so they are linked